oonib/oonibackend.py

```diff
@@ -64,9 +64,9 @@ def getEndpoint(endpoint_config):
 
 def createService(endpoint, role, endpoint_config):
     if role == 'bouncer':
-        factory = ooniBouncer
+        factory = OONIBouncer()
```

Comment:
>> [...] it's perhaps useful to still think of some possible edge case I may not have thought of.

> Reading through the code; will report back soon.

I do not see obvious way in which the discussed change could break existing behavior.